src/content/release-notes.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.38.0
+
+[Date TBD]
+
+This version is compatible with Kubernetes versions 1.31 to 1.33 \
+Okteto Chart release 1.38 is designed to work with [Okteto CLI 3.13.x](https://github.com/okteto/okteto/releases/tag/3.13.0)
+
+### New Features {#new-features-1.38}
+
+- **Enhanced Preview Environments UI**: Dramatically improved loading performance, reducing load times by up to 15x for customers with large numbers of Preview Environments. Added comprehensive filtering options (search, repository, status, owner, and time-based filters) to help teams quickly find the previews they need. Also includes UI refinements such as a personal scope indicator icon, reorganized columns, and abbreviated repository names for better readability <!-- 9761 -->
+- Added read-only mode for Known Hosts configuration, providing enhanced control over SSH host key management <!-- 9726 -->
+
+### Improvements {#improvements-1.38}
```

Comment:
@pchico83 @jLopezbarb We need to include the changes in the build command about the caches. Is it an improvement or a bug fix? Should we complement it with documentation, or a bullet point in the release notes is enough? We have to mention that if caches have to be reused within steps or dockerfiles, they should use the same id in the dockerfile instruction.

Reply:
Yes, we should fix. And I would document our default behavior for the cache id in this doc:
https://www.okteto.com/docs/core/build-service/